libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -103,6 +106,9 @@ class AC_WPNav
     /// get origin using position vector (distance from ekf origin in cm)
     const Vector3f &get_wp_origin() const { return _origin; }
 
+    /// true if origin.z and destination.z are alt-above-terrain, false if alt-above-ekf-origin
```

Comment:
This is an unwarranted assumption; _terrain_alt only tells us whether we've been told to assume z is in terrain alt, not how that determination was made.

Reply:
not sure I understand this actually.. when the set_wp_origin_and_destination was performed, AC_WPNav recorded whether the incoming altitude was a terrain altitude or not and this new accessor just returns that specification..